examples/landsat.ipynb

```diff
@@ -203,7 +200,7 @@
     "    b = (normalize_data(b)).astype(np.uint8)\n",
     "    col, rows = r.shape\n",
     "    img = np.dstack([r, g, b, a]).view(np.uint32).reshape(r.shape)\n",
-    "    return tf.Image(data=img)"
+    "    return tf.Image(data=img[::-1])"
```

Comment:
It's somewhat unfortunate this is needed, in general it is not safe to simply grab the ``.data`` off an xarray ``DataArray`` as is done in the function below because the coordinates could be oriented in any number of ways.

Reply:
Is ::-1 needed for this particular file, but may need to be different for a different file with some other orientation?  If so, can we not compute the "-1" (or "1") from the declared orientation?

Reply:
Yes, I could use the ``orient_array`` utility here instead.

Reply:
Ok, sounds good. Mergeable after that?

Reply:
(A better name for that utility is would be welcome btw).

Reply:
I'll add some unit tests first.

Reply:
reorient_array? convert_array_to_canonical_orientation?  canonicalize_array?  They all seem as bad as each other.

Reply:
Since we like none of them I'll just stick with what we've got.